docs/reference/release-notes/highlights.asciidoc

```diff
@@ -106,6 +106,25 @@ search features like queries and aggregations.
 
 {es-pull}88694[#88694]
 
+[discrete]
+[[synthetic_source_technical_preview]]
+=== Synthetic `_source`
+preview::[]
+When Elasticsearch ingests documents, it creates several data structures that
+enable querying and aggregating the data. One of those data structures is called
+`_source`.  `_source` holds the fields of every document. It uses a lot of space
```

Comment:
`_source` is actually for "fetching" the document, not really "querying" or "aggregating"